debian/rules

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/make -f
 
+override_dh_auto_configure:
+	dh_auto_configure -- -DHAVE_SYSTEMD=1
```

Comment:
I build the package in pbuilder and there systemd is not installed, so this needs to be passed in to install the systemd files